docs/en/latest/config.json

```diff
@@ -146,6 +146,15 @@
       "type": "category",
       "label": "Others",
       "items": [
+        {
+          "type": "category",
+          "label": "discovery",
```

Comment:
Do we need to use the UpperCase? Like Discovery

Reply:
@juzhiyuan 
Updated.